libgalois/include/katana/Properties.h

```diff
@@ -182,6 +171,106 @@ ConstructPropertyViews(const std::vector<arrow::Array*>& arrays) {
       arrays, std::make_index_sequence<std::tuple_size_v<PropTuple>>());
 }
 
+template <typename T>
```

Comment:
Please move all this new code to below anonymous namespace so that it is easier to compare what changed in `AllocateTable`